events/meetups.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Meetup Groups
```

Comment:
Is "Meetup" not a bit brand specific? Might exclude people who aren't using Meetup. What about "Meetups and User Groups"?

Reply:
I thought meetup was the general term for "user group"?

Reply:
But I see what you mean anyway.. I'll commit again and change it to "Meetups and User Groups". Any other changes before I do it?